src/main/flight/pid.c

```diff
@@ -205,6 +205,8 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .auto_profile_cell_count = AUTO_PROFILE_CELL_COUNT_STAY,
         .transient_throttle_limit = 15,
         .profileName = { 0 },
+        .ff_boost = 15,
+        .ff_boost_hz = 20,
```

Comment:
Doesn't these `.ff_boost` and `.ff_boost_hz` go before `.profileName`?

Reply:
If it matters I will be happy to move them up...